src/bacon.rs

```diff
@@ -447,9 +449,15 @@ impl Bacon {
         })
         .expect("failed to create file watcher");
 
-        watcher
-            .watch(PathBuf::from(&locations_file), notify::RecursiveMode::Recursive)
-            .expect("couldn't watch diagnostics file");
+        loop {
+            match watcher.watch(PathBuf::from(&locations_file), notify::RecursiveMode::Recursive) {
+                Ok(_) => break,
+                Err(e) => {
+                    info!("Watcher failed to start");
```

Comment:
I would expand the error a little with what has been failed and why:

`tracing::info!("unable to start .bacon_locations file watcher, retrying in 1 second")`

Reply:
Makes sense, I can make the change tomorrow. I also wonder if we should just directly check for the files existence before this loop and perhaps inspect the actual error condition and fail more gracefully in situations that are probably unrecoverable

Reply:
I see your point, but this works as a quickfix. I'll refactor this part for a later release if needed.